sequins.go

```diff
@@ -114,8 +129,15 @@ func (s *sequins) init() error {
 		s.downloadRateLimitBucket = ratelimit.NewBucketWithRate(float64(maxDownloadBandwidth), maxDownloadBandwidth)
 	}
 
+	// Kick off feature flag cache refresh if GoforitFlagJsonPath configured
+	if s.config.GoforitFlagJsonPath != "" {
+		log.Printf("Enabling Goforit: GoforitFlagJsonPath=%s", s.config.GoforitFlagJsonPath)
```

Comment:
use `%q` instead of `%s`  it will use the correct quotes around strings to make splunk happy.

Reply:
ah cool!  i'll update the other ones i added in this PR